rtloader/test/datadog_agent/datadog_agent.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 	"unsafe"
 
+	"github.com/DataDog/datadog-agent/pkg/trace/obfuscate"
```

Comment:
Please don't link rtloader to any other package from the agent. It's not needed and put an hard dependencies between both. Technically rtloader function could be initialized with any go code, so let's leverage this in our test and keep both of them separate.

Reply:
Ok updated to remove the dependency